CordovaLib/Classes/Private/Plugins/CDVUIWebViewEngine/CDVUIWebViewDelegate.m

```diff
@@ -329,9 +329,14 @@ - (void)webViewDidFinishLoad:(UIWebView*)webView
             break;
 
         case STATE_WAITING_FOR_LOAD_FINISH:
-            if (_loadCount == 1) {
+            // fix call loadRequest multiple times just callback webViewDidFinishLoad once time in iOS 12
+            if (@available(iOS 12.0, *)) {
```

Comment:
What exactly does this line do? 
Does it check for "iOS 12.0"?

Reply:
Yes,check if the system version is "12.0"

Reply:
Doesn't that mean that is already broken again in 12.1?

Reply:
Like @janpio said, you need to test for iOS 12 and greater. Use `IsAtLeastiOSVersion` https://github.com/apache/cordova-ios/blob/1cf92246d5463f519bab76e573078a76d58f9c68/CordovaLib/Classes/Public/CDVAvailability.h#L96

Reply:
@crmo So this fix is just for 12.0 specifically, you mean?

Reply:
The `@available` check includes newer versions that the one specified.